ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -180,7 +182,8 @@ data Cmd blk it flr
     -- smaller than the block's slot number (such that the block is from the
     -- future) and larger or equal to the current slot, and add the block.
   | GetCurrentChain
-  | GetLedgerDB
+  -- TODO(js_ldb): reenable
+  --  GetLedgerDB
```

Comment:
Is there already an issue for this?

Reply:
@jorisdral and the comment below

Reply:
Hmm, unfortunately changing the tests to work with the new `LedgerDB` went under the radar :P

When designing the `LedgerDB` API, the `getLedgerDB` function was removed from the `ChainDB` API, because `LedgerDB` was no longer a pure datastructure. It was replaced by more concrete functions, amongst which: `getCurrentLedger`, `getImmutableLedger`, `getPastLedger`, `getHeaderStateHistory`, `getReadOnlyForkerAtPoint`, `getLedgerTablesAtFor`, `getStatistics`.

Options I can think of for how to restore testing:
* Test (a subset of) the concrete functions by creating a command for each.
* Don't test the concrete functions, but use `TestInternals` to output some pure information about the `LedgerDB`. Maybe we could reconstruct a pure sequence of ledger states from the concrete `LedgerDB` impls (reading tables from disk too), and compare those against the model?

And no, there is no issue for it, but there probably should be

Reply:
Captured the TODO in #1339